src/App.tsx

```diff
@@ -1,61 +1,117 @@
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList/TodoList';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import users from './api/users';
+import todos from './api/todos';
+
+let todoWithUser = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId)?.name || null,
+}));
+
+todoWithUser.sort((todo1, todo2) => todo1.id - todo2.id);
+
+const firstMaxId = todos.sort((todo1, todo2) => todo2.id - todo1.id)[0].id;
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [userForTodo, setUserForTodo] = useState('');
+  const [id, setId] = useState(firstMaxId + 1);
+  const [isInputEntered, setIsInputEntered] = useState(true);
+  const [isUserSelected, setIsUserSelected] = useState(true);
+
+  const noInputMessage = 'Write task, please';
+  const noUserMessage = 'Choose user, please';
```

Comment:
move it outside a component, you don't need to create same variable on every render